src/Modules/Data.lua

```diff
@@ -488,13 +488,28 @@ local function loadJewelFile(jewelTypeName)
 		uncompressedFileAttr.fileName = fileHandle:GetFileName()
 		uncompressedFileAttr.modified = fileHandle:GetFileModifiedTime()
 	end
+
 	fileHandle = NewFileSearch(main.userPath .. jewelTypeName .. ".zip")
 	local compressedFileAttr = { }
 	if fileHandle then
 		compressedFileAttr.fileName = fileHandle:GetFileName()
 		compressedFileAttr.modified = fileHandle:GetFileModifiedTime()
 	end
 
+	fileHandle = NewFileSearch(main.userPath .. jewelTypeName .. ".*.part*")
+	local splitFile = ""
+	if fileHandle then
+		compressedFileAttr.modified = fileHandle:GetFileModifiedTime()
+	end
+	while fileHandle do
+		local file = io.open("Data/TimelessJewelData/" .. fileHandle:GetFileName(), "rb")
+		splitFile = splitFile .. file:read("*all")
```

Comment:
Do we know if the file search always returns them in a ascending order?

Reply:
File searching dispatches to Windows' `FindFirstFile`/`FindNextFile` which explicitly states that the order is arbitrary from the file system and if the application requires a specific order it must sort on its own: [FindNextFileA Remarks](https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-findnextfilea#remarks)

Reply:
I agree this needs to be sorted. Currently it seems like Windows sorts these specific files in the expected order, but if you have a quick way to guarantee this please just push it directly.

Reply:
There's no way of getting a specific order out of the API, you need to gather and order results on the Lua side or do something like speculatively open files sequentially to see if they exist.